jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/EntriesComparator.java

```diff
@@ -40,20 +39,15 @@
    * @return True iff the entries present in the two jar files are the same.
    * @throws IOException if an error happens when reading jar files.
    */
-  @SuppressWarnings("JdkObsolete")
   public static boolean compareEntriesInJars(String jarFile1, String jarFile2) throws IOException {
     Preconditions.checkArgument(jarFile1.endsWith(".jar"), "invalid jar file: " + jarFile1);
     Preconditions.checkArgument(jarFile2.endsWith(".jar"), "invalid jar file: " + jarFile2);
     JarFile jar1 = new JarFile(jarFile1);
     Set<String> jar1Entries = new HashSet<>();
-    for (Enumeration<JarEntry> entries = jar1.entries(); entries.hasMoreElements(); ) {
-      jar1Entries.add(entries.nextElement().getName());
-    }
+    jar1.stream().forEach(entry -> jar1Entries.add(entry.getName()));
```

Comment:
Here I would do something like `Set<String> jar1Entries = jar1.stream().map(Entry::getName).collect(Collectors::toSet)`.  Not sure of exact syntax but something like that.  Same for the other similar cases.

Reply:
Done.